packages/frontend/src/components/FButton.tsx

```diff
@@ -0,0 +1,10 @@
+const Button = ({ label }: { label: string }) => {
+    return (
+      <button className="flex justify-center items-center gap-2 px-7 py-4 border font-montserrat text-lg leading-none rounded-full  
+                        bg-[#575553] text-white border-[#575553]">
+        {label}
+      </button>
+    )
+  }
+  
+  export default Button;
```

Comment:
Note that you are using the same name for the button in both `Fbutton.tsx` and in `TButton.tsx`.

1. It would be better if name the component the same name as the file name.
1. There's no reason to use `export default Button`, in this case, not using `default` is better.
1. In your case,  There is no need to have different components for both transparent and filled buttons,  you can add `isTransparent` or `isFilled` as a property, and use if-statement to handle both cases.